src/Types.js

```diff
@@ -90,6 +90,26 @@ export type EditorRuntime = {
   // External HTML
   canLoadHTML?: () => boolean,
   loadHTML?: () => Promise<?string>,
+
+  /**
+   * Gets array of styles from the service
+   */
+  getStylesAsync: () => Promise<StyleProps[]>,
+  /**
+   * Renames an existing style from the service.
+   * @param oldStyleName
+   * @param newStyleName
+   */
+  renameStyle: (
+    oldStyleName: string,
+    newStyleName: string
+  ) => Promise<StyleProps[]>,
+
+  /**
+   * Remove an existing style from the service.
+   * @param name
+   */
+  removeStyle: (name: string) => boolean,
```

Comment:
Inconsistent API,   Either:

1) All style releated methods resolve to the updated style list.
or 
2) Only getStylesAsync resolves to the full list.   removeStyle resovles to boolean succeess/failure,  saveStyle resolves to new updated style (single item) and renameStyle resolves to updated style.